src/dshould/package.d

```diff
@@ -136,11 +103,11 @@ unittest
     obj.should.be(null)
         .should.throwA!FluentException("Test failed: expected null, but got object.Object");
 
-    obj.should.not.be(obj)
+    obj.should.not.be.same.as(obj)
```

Comment:
move to `basic`, where `same` is defined?

Reply:
Testet hier die Exception.